proto/viam/app/v1/app.proto

```diff
@@ -74,6 +75,7 @@ message RobotPart {
   string location_id = 12 [(tagger.v1.tags) = "bson:\"location_id\" json:\"-\""];
   google.protobuf.Struct robot_config = 5 [(tagger.v1.tags) = "bson:\"config\" json:\"robot_config\""];
   google.protobuf.Timestamp last_access = 6 [(tagger.v1.tags) = "bson:\"last_access\" json:\"last_access\""];
+  google.protobuf.Timestamp created_on = 13 [(tagger.v1.tags) = "bson:\"created_on\" json:\"created_on\""];
```

Comment:
Nit: let's move this to the end to try and keep fields in approximate field number order.  There's no specific reason it needs to be next to last_access.

Reply:
done